HOMEWORKS/Chapter_04/Rasha2/imagesHomework/assets/style/c0.css

```diff
@@ -0,0 +1,34 @@
+#header {
```

Comment:
You set you homework in two places:
- HOMEWORKS/Chapter_04/Rasha2
- HOMEWORKS/Rasha

Please check the right one and delete the other then push after correcting all my comments